pkg/bisect/bisect.go

```diff
@@ -254,14 +254,12 @@ func (env *env) bisect() (*Result, error) {
 	}
 	env.crashReport = testRes.rep
 
-	if len(cfg.Kernel.BaselineConfig) != 0 {
-		testRes1, err := env.minimizeConfig()
-		if err != nil {
-			return nil, err
-		}
-		if testRes1 != nil {
-			testRes = testRes1
-		}
+	testRes1, err := env.minimizeConfig()
```

Comment:
Why not overwrite testRes unconditionally instead of creating testRes1? Should we even continue here if the testRes returned by minimizeConfig is nil?

Reply:
> Should we even continue here if the testRes returned by minimizeConfig is nil?

Yes. If all our config minimization attempts didn't crash, that's not critical and we can proceed.